traffic_ops/traffic_ops_golang/trafficstats/trafficstats.go

```diff
@@ -0,0 +1,691 @@
+package trafficstats
```

Comment:
Is there any unit tests that can be written for this file? 

Like validation on the config or authorized to query endpoint?

Reply:
I could write some negative tests for the format of the request, but I don't handle user permissions in the route handler, and a unit test can't handle a connection to an external InfluxDB server; that's an integration test.